ci/matrix.xml

```diff
@@ -181,16 +181,18 @@
 
   <task id="bootstrap"><![CDATA[
     setJavaVersion $java
-    sbt 'set jsEnv in toolsJS := NodeJSEnv(args = Seq("--max_old_space_size=2048")).value.withSourceMap(false)' \
-        ++$scala toolsJS/test &&
-    sbt 'set jsEnv in toolsJS := NodeJSEnv(args = Seq("--max_old_space_size=2048")).value.withSourceMap(false)' \
+    sbt 'set scalaJSUseRhino in Global := false' \
```

Comment:
It's necessary (as the previous failure showed).
That's because `irJS` needs `zip.js` which does not boot in Rhino.